tests/e2e/logging/test_openshift-logging.py

```diff
@@ -237,7 +239,45 @@ def test_respin_osd_pods_to_verify_logging(self, create_pvc_and_deploymentconfig
         project2_filecount = elasticsearch_pod_obj.exec_cmd_on_pod(
             command=f'es_util --query=project.{project2}.*/_count', out_yaml_format=True
         )
-        logger.info(f'The files in the project2 {project2_filecount}')
+        assert project2_filecount['_shards']['successful'] != 0, f"No files found in project {project2}"
+        logger.info(f'Total number files in project 2 {project2_filecount}')
+
+    @pytest.mark.polarion_id("OCS-665")
+    @tier4
+    def test_create_and_delete_multiple_app_pods_and_verify_logs(self, multi_pvc_factory, dc_pod_factory):
+        """
+        The test is to create and delete multiple app pods in a project
+        to verify in EFK stack
+        """
+
+        # Creates multiple app-pods
+        num_of_pvcs = 1
+        pvc_size = 2
+        pvc_objs = multi_pvc_factory(
+            size=pvc_size, num_of_pvc=num_of_pvcs
+        )
+        dc_pod_objs = [dc_pod_factory(pvc=pvc_obj) for pvc_obj in pvc_objs]
```

Comment:
Test case is to create multiple pods but only one pod and one pvc is being created because num_of_pvcs value is set to 1